lib/util/path-util.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @fileoverview Common helpers for operations on filenames and paths
+ * @author Ian VanSchooten
+ * @copyright 2016 Ian VanSchooten. All rights reserved.
+ * See LICENSE in root directory for full license.
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var path = require("path"),
+    isAbsolute = require("path-is-absolute");
+
+//------------------------------------------------------------------------------
+// Private
+//------------------------------------------------------------------------------
+
+/**
+ * Replace Windows with Unix style paths and remove ./ prefix and any
+ * unnecessary "../"
```

Comment:
This description doesn't seem correct. You're not removing those parts, you're resolving them, correct?